src/routes.js

```diff
@@ -1,8 +1,13 @@
 const express = require('express');
 const app = express();
+const Client = require('../src/client');
 
-app.get('/bars', (req, res) => {
-  res.json({results: []});
+app.get('/bars', async (req, res) => {
+  const client = new Client;
+
```

Comment:
Will this be the route we need to send data to from the frontend?

Reply:
This was just temp to get visability on the actual call

Reply:
ok cool